Makefile

```
@@ -85,13 +85,18 @@ infra-update-app-database-roles:
 	@:$(call check_defined, ENVIRONMENT, the name of the application environment e.g. "prod" or "staging")
 	./bin/create-or-update-database-roles.sh $(APP_NAME) $(ENVIRONMENT)
 
-
 infra-update-app-service:
 	# APP_NAME has a default value defined above, but check anyways in case the default is ever removed
 	@:$(call check_defined, APP_NAME, the name of subdirectory of /infra that holds the application's infrastructure code)
 	@:$(call check_defined, ENVIRONMENT, the name of the application environment e.g. "prod" or "staging")
 	./bin/terraform-init-and-apply.sh infra/$(APP_NAME)/service $(ENVIRONMENT)
 
+infra-monitoring-secrets:
```

nit: since this is a configuration step, let's add the word `configure` to be consistent with the other configuration steps
```suggestion
infra-configure-monitoring-secrets:
```

---

done